Xamarin.Forms.Core/Picker.cs

```diff
@@ -46,6 +56,16 @@ public string Title
 			set { SetValue(TitleProperty, value); }
 		}
 
+		public IEnumerable<object> ItemsSource {
```

Comment:
Why is the BP `ItemsSourceProperty` an `IEnumerable` and the property an `IEnumerable<object>` ?